lib/msal-core/src/Constants.ts

```diff
@@ -120,6 +120,11 @@ export const PromptState = {
 	NONE: "none",
 };
 
+/**
+ * Key-Value type to support queryParams and extraQueryParams
+ */
+export type QPDict = {[key: string]: string};
```

Comment:
We should try to separate string constants and types that we want to export. Would a separate file for types be better?

Reply:
May be. If we have more of these. Let me make a call once I review how many generic types we have in our code to separate them from Constants.ts file.

Reply:
Done, introduced a new file for 'MsalTypes'